### PR TITLE
Test runner switched to Pytest

### PR DESCRIPTION
**Pull request by panasevychol**
Test runner switched to Pytest

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -125,7 +125,7 @@ pylint:
 
 .PHONY: test
 test:
-	python -m unittest discover -s .
+	pytest tests
```
Comment (5A11) [inline Makefile:128]: Please add `--doctest-modules` option to enforce doctests.

Comment (5A11) [reply Makefile:128]: Also please add coverage reports. 
You will need to add the correct plugin (`pytest-cov` ), specify the correct options here (e.g. `--cov` and `--cov-report=`, ...), and place the necessary configs in the repo (e.g. `.coveragerc`). See the aea framework for reference.

**Review by 5A11 — APPROVED** [review]
LGTM

**Review by MissingNO57 — APPROVED** [review]
LGTM

**Pull request merged**